apps/src/dance/types.ts

```diff
@@ -60,3 +60,7 @@ export type MinMax = {
   minIndividualScore: number;
   maxTotalScore: number;
 };
+
+export interface AiFieldValue extends GeneratedEffect {
+  inputs: string[];
```

Comment:
Is it worth capturing the "good" effect that is also stored in the field value?

Reply:
Well since `AiFieldValue` extends `GeneratedEffect`, the interface should contain both `inputs` and the effect values - is that what you're referring to? Or am I missing something?

Reply:
Oh you're right.  All good!